go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/yahoojapan/authorization-proxy
+module github.com/yahoojapan/authorization-proxy/2
```

Comment:
```suggestion
module github.com/yahoojapan/authorization-proxy/v2
```
should be `v2`, just `2` does not work.
The `v` prefix is required for golang to treat it as version.
https://research.swtch.com/vgo-import

Reply:
flow:
1. merge this PR
1. `2.0.1` released by pipeline
1. tag `v2.0.1` and release manually

Reply:
OK, I  fixed it according to this.
9aee247